### PR TITLE
Added Keras benchmark script to BenchmarkAI repo

### PR DESCRIPTION
**Pull request by karan6181**
- Added Keras benchmark script to BenchmarkAI repo
- Added `cifar10_resnet` and `lstm_text_generation_nietzsche` test for benchmarkAI

@sandeep-krishnamurthy @roywei @kalyc

### CODE REVIEW

**Review by sandeep-krishnamurthy — APPROVED** [review]
Please add reference link to awslabs/keras-apache-mxnet benchmark module.

**Review by roywei — APPROVED** [review]

**Review by kalyc — APPROVED** [review]
Added few comments, please address them before merging

File: keras_benchmark/scripts/benchmark_resnet.py [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+            index = index + 1
+            current_index = current_index + 1
+        except:
+            print("Ignore image {}".format(train_images[current_index]))
+            current_index = current_index + 1
+
+    return B, keras.utils.to_categorical(L, num_classes)
+
+
+# Prepare Model
```
Comment (kalyc) [inline keras_benchmark/scripts/benchmark_resnet.py:189]: move this to README file - any reason why this has been included with the code?

File: keras_benchmark/scripts/benchmark_resnet.py [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+    # Normalize data.
+    x_train = x_train.astype('float32') / 255
+    x_test = x_test.astype('float32') / 255
+
+    # If subtract pixel mean is enabled
+    if subtract_pixel_mean:
+        x_train_mean = np.mean(x_train, axis=0)
+        x_train -= x_train_mean
+        x_test -= x_train_mean
+    #x_train = x_train[:200]
```
Comment (kalyc) [inline keras_benchmark/scripts/benchmark_resnet.py]: please remove commented code

File: keras_benchmark/scripts/benchmark_resnet.py (hunk 1 above)
Comment (karan6181) [reply keras_benchmark/scripts/benchmark_resnet.py:189]: These results are from official TensorFlow benchmark repo and the data is from original papers and not from our benchmark script. So, didn't move to the README file.

File: keras_benchmark/scripts/benchmark_resnet.py (hunk 2 above)
Comment (karan6181) [reply keras_benchmark/scripts/benchmark_resnet.py]: Done. Thanks.

**Pull request merged**